lib/ex_doc/formatter/html.ex

```diff
@@ -543,4 +552,33 @@ defmodule ExDoc.Formatter.HTML do
         {base, opts[:filename] || text_to_id(Path.rootname(base))}
     end)
   end
+
+  defp sectionize(list), do: sectionize(list, [])
```

Comment:
Let's move this function to the `Markdown` module, WDYT?

Reply:
I thought on doing that, wasn't sure about the intention of such a module.